README.md

```diff
@@ -1,35 +1,33 @@
 # The SpatialOS GDK for Unity (alpha)
 
-The SpatialOS Game Development Kit (GDK) for Unity enables game developers to build multiplayer [Unity](http://unity3d.com) games with [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). It will be made up of three parts:
+The SpatialOS Game Development Kit (GDK) for Unity allows you to quickly and easily build and host Unity multiplayer games. These games can use multiple server-side game engines across one seamless world to create new kinds of gameplay.
 
-* the GDK Core, a performant, data-driven [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos) multiplayer integration.
+The GDK is composed of three layers:
 
-* a library of solutions for hard or common networked game development problems via Feature Modules.
+**The GDK Core:** a performant, data-oriented integration with our cloud platform SpatialOS, based on the familiar Unity-native workflows.
 
-* the example game, which both tests and demonstrates the Feature Modules; for use as a starting point or resource for game development.
+**The GDK Feature Modules:** a library of solutions for hard or common networked game development problems, such as Character Movement and Shooting.
 
-This is an alpha release of the GDK Core which is the foundation of the product. To find out more see the [GDK blogpost](https://improbable.io/games/blog/unity-gdk-our-first-steps).
+**The First Person Shooter (FPS) Starter Project:** a starting sandbox project for the GDK that enables you and your friends to experience the true scale of SpatialOS, providing a solid foundation for entirely new games.
```

Comment:
another comment was that this doesn't show "true scale", but should be something along the lines of "a taste of the scale that SpatialOS provides"

Reply:
Same as above... :-)